### DISCUSSION TITLE
new Kimai 2.x and php 8.0

### DISCUSSION THREAD

**Discussion by BeckeBauer** [kimai/kimai#3738] [Questions & Answers] [answered]
Currently, I run Kimai on a Synology Server. It is announced that the new Kimai 2.x will need php 8.1 to run. Unfortunately,  Synology does not support 8.1 yet (it is unknown if and when this will happen). I am reluctant so far to deploy Kimai using Docker (have not checked if it is even possible with the new version). So, I am wondering if there is any chance that Kimai 2.x would work with php 8.0 as well?

**Comment by kevinpapst** [comment, marked as answer]
Most libraries that are used by Kimai require PHP 8.1. The new version itself uses many "8.1 only" features as well.
So the clear is answer is: no.

So  Synology, a huge company and one of the leaders in home NAS technology, is still not pushing a PHP version that was released over a year ago https://www.php.net/supported-versions.php ?

A solution for you could be docker or the cloud, until Synology decides to ship a more recent PHP version.

Sorry for not having more comforting feedback 😞 

I find [this post](https://community.synology.com/enu/forum/1/post/156517) rather interesting:

>  You only need to look at the up coming new features in DSM 7.2 to see where Synology's priorities lie and it's not with having their products being web servers.

I would consider switching technology if PHP is important for you.

**Comment by BeckeBauer** [comment]
Thanks for the (not really unexpected) clarification.

It is really embarrassing that Synology is not able to ship an up-to-date version of php more than 1 year after its release - but I am not ready to change hardware yet. Maybe, I will give Docker a try instead
